packages/page-staking/src/Performance/useAbftScores.tsx

```diff
@@ -2,12 +2,12 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import type { Option, Struct, Vec } from '@polkadot/types';
-import type { Hash, SessionIndex } from '@polkadot/types/interfaces';
+import type { SessionIndex } from '@polkadot/types/interfaces';
 import type { u16, u32 } from '@polkadot/types-codec';
 
 import { useEffect, useState } from 'react';
 
-import { getCommitteeManagement } from '@polkadot/react-api';
+import { getCommitteeManagement } from '@polkadot/react-api/getCommitteeManagement';
```

Comment:
Is this change necessary? If `getCommitteeManagement` could be imported from `@polkadot/react-api`, then it should - nested imports are probably non-stable.